src/Silences/Types.elm

```diff
@@ -90,3 +32,20 @@ nullDuration =
 nullTime : Time
 nullTime =
     Utils.Date.fromTime 0
+
+
+type alias Silence =
+    { id : SilenceId
+    , createdBy : String
+    , comment : String
+    , startsAt : Time
+    , endsAt : Time
+    , duration : Duration
+    , updatedAt : Time
+    , matchers : List Matcher
+    , silencedAlertGroups : ApiData (List AlertGroup)
+    }
+
+
+type alias SilenceId =
```

Comment:
This will not create a new type, so can be substituted with `String`. If you want it to be typechecked, you can do `type SilenceId  = SilenceId String` and then write a decoder that fails for invalid silence ids.